packages/web-ui/src/ds/atoms/Icons/index.tsx

```diff
@@ -2,15 +2,23 @@ import {
   ChevronDown,
   ChevronRight,
   Copy,
+  EllipsisVertical,
   File,
   FolderClosed,
   FolderOpen,
   type LucideIcon,
 } from 'lucide-react'
 
+import { type TextColor } from '$ui/ds/tokens'
+
 import { LatitudeLogo, LatitudeLogoMonochrome } from './custom-icons'
 
 export type Icon = LucideIcon
+export type IconProps = {
+  color?: TextColor
+  widthClass?: string
+  heightClass?: string
+}
```

Comment:
What about using just `className`?

Reply:
I prefer concrete props over free for all put whatever class you want